tests/spread/core24/architectures/task.yaml

```diff
@@ -29,15 +31,23 @@ execute: |
 
   # if the environmental variable file exists, then call snapcraft with the environmental variables
   if [[ -e "environmental-variables.txt" ]]; then
+    # If this next line fails, delete it and replace it with the one following it
```

Comment:
yeah I thought about doing this but I thought it might get too confusing with "comments that are for shellcheck annotations" and "comments that are commenting out the real test". In the end I think this current status is fine, shouldn't be like this for too long anyway. Thanks for the review.